com.unity.shadergraph/Documentation~/Linear-Blend-Skinning-Node.md

```diff
@@ -0,0 +1,15 @@
+# Linear Blend Skinning Node
+
+## Description
+
+This node lets you apply Linear Blend Vertex Skinning, and only works with the [DOTS Hybrid Renderer](https://docs.unity3d.com/Packages/com.unity.rendering.hybrid@latest/). You must provide skinned matrices in the `_SkinMatrices` buffer. The node uses the `_SkinMatrixIndex` property to calculate where the matrices associated with the current mesh are located in the `_SkinMatrices` buffer.
```

Comment:
Just going to echo what I noted for the Compute Deformation Node, if there's any specific external documentation for DOTS Hybrid that provides more context for this data and the use cases, that would be good to add here.